internal/logger/logger.go

```diff
@@ -103,10 +103,10 @@ func InitLogFile(legacyLogConfig config.LogConfig, newLogConfig cfg.LoggingConfi
 // init initializes the logger factory to use stdout and stderr.
 func init() {
 	defaultLoggerFactory = &loggerFactory{
-		file:            nil,
-		format:          defaultFormat,
-		level:           config.INFO, // setting log level to INFO by default
-		logRotateConfig: config.DefaultLogRotateConfig(),
+		file:      nil,
+		format:    defaultFormat,
+		level:     config.INFO, // setting log level to INFO by default
```

Comment:
Shall we move these constants to new config package and use from there?

Reply:
Currently, newConfig has dependency on old config. To avoid cyclic dependency, I'll wait for https://github.com/GoogleCloudPlatform/gcsfuse/pull/2258 to be submitted where you remove that dependency.